tokio-util/tests/io_reader_stream.rs

```diff
@@ -0,0 +1,66 @@
+#![warn(rust_2018_idioms)]
+#![cfg(feature = "full")]
```

Comment:
Do these cfg attributes actually work for files in the `tests/` dir? IIRC, the crate's feature flags are only passed through to unit tests in `src/`. It might be worth double-checking that these tests are actually run?

Reply:
No idea. Some other tests have it too.